src/main/java/org/broadinstitute/hellbender/utils/codecs/table/TableFeature.java

```diff
@@ -1,26 +1,27 @@
 package org.broadinstitute.hellbender.utils.codecs.table;
 
 
+import htsjdk.samtools.util.Locatable;
 import htsjdk.tribble.Feature;
-import org.broadinstitute.hellbender.utils.GenomeLoc;
-import org.broadinstitute.hellbender.utils.Utils;
 
 import java.util.List;
 
+import static org.broadinstitute.hellbender.utils.Utils.join;
+
 /**
  * A feature representing a single row out of a text table
  */
-public class TableFeature implements Feature {
-    // stores the values for the columns seperated out
+public final class TableFeature implements Feature {
```

Comment:
Document the fact that stop positions not explicitly specified in the interval column of records will be filled in with `Integer.MAX_VALUE`